tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -75,7 +75,7 @@ apiVersion: heat.openstack.org/v1beta1
 kind: HeatCfnAPI
 metadata:
   finalizers:
-    - HeatCfnAPI
+    - openstack.org/heatcfnaPI
```

Comment:
Wondering if `heatcfnaPI` might have wrong case

Reply:
right thats wrong, thanks for the pointer

Reply:
fixed